INSTALL.md

```diff
@@ -0,0 +1,360 @@
+# Basic installation instructions
+
+Stable version installation on Unix-like systems (e.g. GNU/Linux or *BSD):
+* Check your OS repository for already built packages.
+* Check https://celestia.space/download.html.
+* Check https://bintray.com/celestia if it contains packages for your system.
+
+Stable version installation on Windows and OSX:
+* Check https://celestia.space/download.html.
+
+Development snapshots installation on Unix-like systems:
+### On Debian 10 and derived systems:
+
+```
+curl https://download.opensuse.org/repositories/home:/munix9:/unstable/Debian_10/Release.key
+sudo apt-key add -echo "deb https://download.opensuse.org/repositories/home:/munix9:/unstable/Debian_10/ ./" 
+sudo tee /etc/apt/sources.list.d/celestia-obs.list
+sudo apt update && sudo apt install celestia
```

Comment:
this block is brocken, it should be:
```
  curl https://download.opensuse.org/repositories/home:/munix9:/unstable/Debian_10/Release.key | sudo apt-key add -
  echo "deb https://download.opensuse.org/repositories/home:/munix9:/unstable/Debian_10/ ./" | sudo tee /etc/apt/sources.list.d/celestia-obs.list
  sudo apt update && sudo apt install celestia
```

Reply:
you still have 2 lines written as one: `sudo apt-key add -echo ...`. `echo` should on the next line